File: csharp/DVSClient/DVSClient/server/address/suggestions/RestApiSuggestionsFormatResponse.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Suggestions
{
    public class RestApiSuggestionsFormatResponse : RestApiResponse
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }

        [JsonProperty("result")]
        public RestApiSuggestionsFormatResult? Result { get; set; }
    }
}

File: typescript/src/server/address/suggestions/restApiSuggestionsFormatResponse.ts
import { RestApiResponse } from "../../restApiResponse";
import { RestApiResponseError } from "../../restApiResponseError";
import { RestApiSuggestionsFormatResult } from "./restApiSuggestionsFormatResult";

export type RestApiSuggestionsFormatResponse = RestApiResponse & {
    error?: RestApiResponseError;
    result?: RestApiSuggestionsFormatResult;
};